lib/raygun.ts

```diff
@@ -248,6 +269,8 @@ class Raygun {
           : message;
     }
 
+    message.details.correlationId = correlationId;
```

Comment:
Should the message always have a `correlationId` or only if the `apmBridge` has been used?

Reply:
Good point, might as well save the bytes if the `apmBridge` isn't present :)